content/TruenasApps/EnterpriseApps/AsigraApp.md

```diff
@@ -0,0 +1,280 @@
+
+During backups, the DS-Client extracts changed data, compresses it, and encrypts the items specified for backup.
+Only new or modified data is backed up. This accelerates the backup transmission time.
+Backed-up data is transmitted to the secure off-site data center hosting the DS-System vault over the Internet, intranet, or a direct dial-up connection.
+After clearing security measures for the DS-Client, restores are performed on demand, through the same DS-Client.
+
+The TrueNAS Asigra DS-System app is the vault. An agentless DS-Client backs up data in the vault.
+The DS-Operator is the management software for the DS-System.
+
+After installing the Asigra DS-System app, register an account with Asigra if you do not already have one.
+Refer to communications from Asigra for the license and the link to the DS-Operator download.
+See [Setting Up Asigra in the DS Operator](#setting-up-asigra-in-the-ds-operator) below for the next steps after installing the Asigra DS-System app.
+
+{{< include file="/static/includes/AppsUnversioned.md" >}}
+
+## Before You Begin
+Before you install the Asigra app:
+
+* Read the [Asigra DS-System User Guide](https://asigra-documentation.s3.amazonaws.com/Guides/Cloud+Backup/v14.2/DS-System_User_Guide.pdf).
+  
+  For information on the DS-License Server, see Section 2.1 in the Asigra DS-System User Guide.
+
+  For information on creating customer accounts and DS-Client, see Section 3.1 in the Asigra DS-System User Guide.
+
+  For more information on DS-Client accounts, see Section 4.1 in the Asigra DS-System User Guide.
+    
+{{< include file="/static/includes/apps/BeforeYouBeginStableApps.md" >}}
+
+{{< include file="/static/includes/apps/BeforeYouBeginRunAsUser.md" >}}
+
+<div style="margin-left: 33px">{{< trueimage src="/images/SCALE/Apps/AsigraAppDetailsScreen.png" alt="Asigra App Details Screen" id="Asigra App Details Screen" >}}</div>
+
+{{< include file="/static/includes/apps/BeforeYouBeginAddNewAppUser.md" >}}
+<div style="margin-left: 33px">The Asigra has two run-as user IDs, <b>0</b> for the root users and <b>999</b> for the postgres data user.
+The Asigra and TrueNAS default user <b>oper</b> serves as the DS-Operator user with administration roles.
+Log into the DS-Operator GUI with this user and the password entered as the **Operator Password** in the wizard after deploying the app.
+You do not need to add the **oper** user in TrueNAS as a user account.
+</div>
+
+{{< include file="/static/includes/apps/BeforeYouBeginAddAppDatasets.md" >}}
+
+<div style="margin-left: 33px"><a href="https://www.truenas.com/docs/scale/scaletutorials/datasets/datasetsscale/">Create the dataset(s)</a> before beginning the app installation process.
+Asigra expects two host path dataset storage volumes, <b>data</b> and <b>postgres_data</b>.
+You can create more datasets if you want additional storage volumes for the app, but these are not required and added after installing the app.</div>
```

Comment:
"**can be** added after installing" sounds better? Added sounds strange to my ears.

Reply:
Content changes in the next commit.